src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSizeTest.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Formats.Tar;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+[Trait("Category", "SdkContent")]
+public class ArtifactsSizeTest : SmokeTests
+{
+    private static readonly string BaselineFilePath = BaselineHelper.GetBaselineFilePath($"ArtifactsSizes/{Config.TargetRid}.txt");
+    private static readonly Dictionary<string, long> BaselineFileContent = new Dictionary<string, long>();
+    private static readonly Regex BuildVersionPattern = new(@"\b\d+\.\d+\.\d+[-@](alpha|preview|rc|rtm)\.\d(\.\d+\.\d+)?\b");
+    private const int SizeThreshold = 25;
+
+
+    public ArtifactsSizeTest(ITestOutputHelper outputHelper) : base(outputHelper)
+    {
+        if(File.Exists(BaselineFilePath))
+        {
+            string[] baselineFileContent = File.ReadAllLines(BaselineFilePath);
+            foreach(string entry in baselineFileContent)
```

Comment:
nit: While this is valid syntax, the convention is to have a space. When in doubt, following the coding convention of the file being edited is the way to go.

```suggestion
        if (File.Exists(BaselineFilePath))
        {
            string[] baselineFileContent = File.ReadAllLines(BaselineFilePath);
            foreach (string entry in baselineFileContent)
```

Reply:
Addressed in this [commit](https://github.com/dotnet/installer/pull/17233/commits/ed23432b0722b0e225d5eeed5dc3029ba035003a).